src/Network/Network/FirewallPolicy/PolicySettings/AzureApplicationGatewayFirewallPolicySetting.cs

```diff
@@ -72,6 +72,11 @@ public class AzureApplicationGatewayFirewallPolicySetting : NetworkBaseCmdlet
         [ValidateNotNullOrEmpty]
         public PSApplicationGatewayFirewallPolicyLogScrubbingConfiguration LogScrubbing { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "Web Application Firewall JavaScript Challenge Cookie Expiration time in minutes.")]
+        [ValidateNotNullOrEmpty]
```

Comment:
Is this really mandatory field?

Reply:
it’s not, mandatory is false

Reply:
Then why can it not be null? May be I don't understand this "ValidateNotNullOrEmpty"

Reply:
this is just just validating when the parameter is there, it's not null or empty